svthalia/concrexit
# Add token info dates

Closes #3100.



### Summary
Adds a column that adds timestamps to the authorised-apps page for
-  creation
- update (= last use?)
- expiration (do we use this?)

### How to test

1. Set up OAuth (how??)
2. Check that the dates are correct

Maybe it's easier to test this on staging??
